package.json

```diff
@@ -590,6 +590,12 @@
 					"items": {
 						"type": "string"
 					}
+				},
+				"dart.selectDeviceOnConnect": {
```

Comment:
I think maybe we should prefix it with `flutter` to match the other Flutter settings. This makes it convenient to find all the flutter settings in the options (in hind-sight, maybe they should be `flutter.` instead of `dart.`... maybe something to change in future!).